src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import cn from "classnames"
+import s from "./Footer.module.css"
+import Container from "../Container/Container"
+
+interface IFooterProps {
+  className?: string
+}
+
+function Footer({ className }: IFooterProps) {
```

Comment:
Just an FYI; React provides a type for functional components:
```typescript
const Footer: React.FC<IFooterProps> = ({ className }) => {
  ...
};
```

One of the advantages of using it is that your return value gets also typed.

Reply:
yeah. that's nice option. shall use it